Plugins~/Src/MeshSyncClientBlender/msblenUtils.cpp

```diff
@@ -3,157 +3,184 @@
 #include "msblenBinder.h"
 #include "msblenUtils.h"
 
+#include "BlenderPyObjects/BlenderPyScene.h"
+
 namespace bl = blender;
 namespace msblenUtils {
-std::string get_name(const Material *obj)
-{
-    std::string ret;
-    if (obj) {
-        ret.assign(obj->id.name + 2);
-        mu::SanitizeNodeName(ret);
+    std::string get_name(const Material* obj)
+    {
+        std::string ret;
+        if (obj) {
+            ret.assign(obj->id.name + 2);
+            mu::SanitizeNodeName(ret);
+        }
+        return ret;
```

Comment:
Is there any change in this file apart from the style changes? If not, can it be reverted until we get a linter?